pttrack/views.py

```diff
@@ -372,19 +372,46 @@ def choose_clintype(request):
                            'choice_key': RADIO_CHOICE_KEY})
 
 
-def action_required_patients(request):
-    ai_list = mymodels.ActionItem.objects.filter(
-        due_date__lte=django.utils.timezone.now().today())
+def home_page(request):
+    active_provider_type = get_object_or_404(mymodels.ProviderType,
+                                             pk=request.session['clintype_pk'])
+    if active_provider_type.signs_charts:
+        workup_list = mymodels.Workup.objects.all()
+        pt_list = list(set([wu.patient for wu in workup_list if wu.signer is None]))
+
+        def byName_key(patient):
+                return patient.last_name
+
+        pt_list.sort(key = byName_key)
+        pagetitle = "Unsigned Workups"
+
+    elif active_provider_type.short_name == "Coordinator":
+        ai_list = mymodels.ActionItem.objects.filter(
+            due_date__lte=django.utils.timezone.now().today())
+
+        # if the AI is marked as done, it doesn't contribute to the pt being on
+        # the list.
+        pt_list = list(set([ai.patient for ai in ai_list if not ai.done()]))
+        pagetitle = "Action Required"
+
+    else:
+        patient_list = mymodels.Patient.objects.all().order_by('last_name')
+        pt_list = list(patient_list)
+        pagetitle = "Patient List (Click Intake for New Patient)"
```

Comment:
This will show up at the top, right? That banner is pretty big. I was thinking maybe that instead of having "click intake for a new pt" there, we set up the template so that if the list is empty it puts a small grey "there aren't any patients without notes... click here to create a new patient" where we would normally see the patient.

I'm thinking that either the "default" filter or an {% if len(list) == 0 %} might be useful for this...